src/AcmTable/AcmTable.tsx

```diff
@@ -345,18 +348,20 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
         direction: SortByDirection.asc,
     }
     const initialSort = props.initialSort || defaultSort
+    const initialSearch = props.initialSearch || ''
+    const initialFilters = props.initialFilters || {}
 
     // State that can come from context or component state (perPage)
     const [statePerPage, stateSetPerPage] = useState(DEFAULT_ITEMS_PER_PAGE)
     const { perPage: contextPerPage, setPerPage: contextSetPerPage } = useContext(AcmTablePaginationContext)
-    const perPage = contextPerPage || statePerPage
+    const perPage = props.perPage || contextPerPage || statePerPage
```

Comment:
This might lock it to props.perPage
instead maybe
  const [statePerPage, stateSetPerPage] = useState(props.perPage ?? DEFAULT_ITEMS_PER_PAGE)